### DISCUSSION TITLE
Set path to singularity executable/binary

### DISCUSSION THREAD

**Discussion by Fyodir** [nextflow-io/nextflow#3985] [Q&A]
Hi,

I am currently attempting to move my functioning nextflow pipeline to an auto-scaling SLURM cluster on the cloud.

I am required to install singularity/apptainer in a designated drive accessible by all of the compute nodes. However, as singularity/apptainer is not being installed in the default location such as ona  single machine, or via a package manager that makes singularity/apptainer available on all compute nodes. I would like to provide a path for singularity/apptainer to nextflow to use when looking for the singularity/apptainer image.

Does nextflow provide this functionality? As I cannot seem to find a solution in the nextflow documentation.

**Comment by pditommaso** [comment]
Hi, nextflow just runs the `singularity` binary, which is expected to be accessible in a directory on the system `PATH`. 

The fact that it is not installed to a default location, should not prevent modifying the `$PATH` via `.bash_profile` or `.bashrc` files
